Source/Analytics/Web/src/components/MapOverview.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+
+function ListOfHealthRisks({clusters,healthRisks, icons}){
+    
+    var listItem = healthRisks.map(function(healthRisk, i){
+        var cl = icons[i];
+        var clo = clusters[i];
+        return <li className="map-overview-list-item"><div className={clo}></div> <i class={cl} ></i> : {healthRisk}</li>
```

Comment:
class in ``` <i class={cl} ></i>``` should be className

Reply:
Here also use key in the list for React not to complain. You can use the variable i in the map function